outage/2024-10-09-ota-lite.md

```diff
@@ -0,0 +1,57 @@
+## OTA Lite Outage
```

Comment:
api.fio refers to about 2x the functionality that just ota-lite does. I've tried to update the intro paragraph to make this nicer

Reply:
> api.fio refers to about 2x the functionality that just ota-lite does

That's fine.
You may describe what ota-lite is inside the document, just as you did.

On the title level I recommend using either `2024-10-09 Foundries.io resources outage` or `2024-10-09 api.foundries.io resources outage`.
I much prefer the first title, as the impact was much wider than just the API. For example, a part of the CI was also not working properly as well as a major part of the Web UI.

If you look at the outage history, this is what we used in similar past incidents.
Also, from the user perspective, it's easier to grasp/find this outage with a more public-facing resource name.

Reply:
You have the right for opinion; I respect that.

Could you use `2024-10-09 api.foundries.io server outage` (or smth similar) instead of `OTA Lite Outage` then?
IIUC this status is aimed at our users and not us. Let's keep our internal term `OTA Lite` for ourselves.